functions/v2/test/index.test.js

```diff
@@ -110,13 +62,18 @@ describe('functions_cloudevent_storage', () => {
         name: 'my-file.txt',
       },
     };
-    const response = await invocation(PORT, event);
-    ffProc.kill();
-
-    const output = await ffProcHandler;
-    console.log();
+    const server = getTestServer('helloGCS');
+    await supertest(server)
+      .post('/')
+      .send(event)
+      .set('Content-Type', 'application/json')
+      .expect(204);
+
+    const output = console.log
+      .getCalls()
+      .map(x => x.args[0])
```

Comment:
Sinon has assertions for particular calls. Can we use something like [this](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/679979ff03f1793626a12b0aacd6f15bbf4ba3fd/functions/firebase/test/index.test.js#L67-L70)?

Reply:
Done